core/src/main/java/org/keycloak/crypto/SecretSignatureVerifierContext.java

```diff
@@ -0,0 +1,39 @@
+package org.keycloak.crypto;
+
+import org.keycloak.common.VerificationException;
+
+import javax.crypto.Mac;
+import java.security.MessageDigest;
+
+public class SecretSignatureVerifierContext implements SignatureVerifierContext {
```

Comment:
`SymmetricSignatureVerifierContext`?

Reply:
Symmetric is more an encryption thing. Going for MacSignature..